nucliadb/VERSION

```diff
@@ -0,0 +1 @@
+../VERSION
```

Comment:
Why do we need a top level VERSION file? If we had Rust crates more organized, we'd have a folder per package with it's own version file inside

Reply:
We use this VERION as the main version for all of nucliadb.

Everything else is a dependency

It seemed to me that it should stay at the top level.